src/Intl/Idn/composer.json

```diff
@@ -10,15 +10,20 @@
             "name": "Laurent Bassin",
             "email": "laurent@bassin.info"
         },
+        {
+            "name": "Trevor Rowbotham",
+            "email": "trevor.rowbotham@pm.me"
+        }
         {
             "name": "Symfony Community",
             "homepage": "https://symfony.com/contributors"
         }
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/polyfill-mbstring": "^1.3",
-        "symfony/polyfill-php72": "^1.10"
+        "symfony/polyfill-intl-normalizer": "^1.10",
+        "symfony/polyfill-mbstring": "^1.10",
+        "symfony/polyfill-php70": "^1.10"
```

Comment:
we don't need the polyfill for php72 anymore?

Reply:
Initially we did not, but when I started using `mb_chr()` I brought in polyfill-mbstring. Given that all we need is `mb_chr()` it probably makes sense to just use polyfill-php72 since it is a lighter weight polyfill and drop polyfill-mbstring. Any objections?

Reply:
none - the lightest the better

Reply:
Well, if we need a PHP 7.2+ method of mbstring, we need `polyfill-php72`. Otherwise, it will cause issues with projects using PHP 7.1 + ext-intl and replacing the mbstring polyfill due to that. That's precisely why `polyfill-php72` is also implementing it.